.travis.yml

```diff
@@ -9,10 +11,16 @@ env:
 cache:
   directories:
     - "node_modules"
+before_install:
+  - openssl aes-256-cbc -K $encrypted_1688a3f2e0a8_key -iv $encrypted_1688a3f2e0a8_iv -in test/monster_rsa.enc -out ~/.ssh/monster_rsa -d
```

Comment:
Minor: any reason why this cannot be inside of `setup_test_environment`?

Reply:
Actually I tried this and it simply didn't work for some reason. Plus that line was automatically added to this step by Travis CLI, according to this: https://docs.travis-ci.com/user/encrypting-files/